tests/tests/block-stale-reads.sh

```diff
@@ -6,56 +6,21 @@
 # we use triggers to exercise how readers encounter stale blocks.
 #
 
-t_require_commands touch setfattr getfattr
-
-inc_wrap_fs_nr()
-{
-	local nr="$(($1 + 1))"
-
-	if [ "$nr" == "$T_NR_MOUNTS" ]; then
-		nr=0
-	fi
-
-	echo $nr
-}
-
-GETFATTR="getfattr --absolute-names"
-SETFATTR="setfattr"
-
 echo "== create shared test file"
-touch "$T_D0/file"
-$SETFATTR -n user.xat -v 0 "$T_D0/file"
 
 #
-# Trigger retries in the block cache as we bounce xattr values around
-# between sequential pairs of mounts.  This is a little silly because if
-# either of the mounts are the server then they'll almost certaily have
-# their trigger fired prematurely by message handling btree calls while
-# working with the t_ helpers long before we work with the xattrs.  But
-# the block cache stale retry path is still being exercised.
-#
+# Trigger retries in the block cache by calling scoutfs df
+# which in turn will call scoutfs_ioctl_alloc_detail. This
+# is guaranteed to exist, which will force block cache reads.
+
 echo "== set and get xattrs between mount pairs while retrying"
 set_nr=0
```

Comment:
The set and get names were from setattr and getattr, which we're not doing anymore.   This should be just 'nr' which the tests tend to use as the mount nr.  It matches using M0 as the df path later.